### ISSUE TITLE
Tileset have some problems with animation operations

### ISSUE THREAD

**Issue by CodeLazier** [godotengine/godot#63102]
### Godot version

4.0 a12

### System information

win10 vulkan desktop gtx1060 6G

### Issue description

[tsa.webm](https://user-images.githubusercontent.com/14953625/179390508-dcafd9f0-fd60-43da-aa8e-416340c807a9.webm)

Tilesets have some problems with animation operations,But some issues require feedback.
1. Separation unit should not be px,It is clearly misleading.
2. Some operations require order, otherwise nothing will happen. some hints should be added to this.
3. Clear Array nothing happened.
4. Click Frames is a certain probability crash directly and exit to the desktop,  after separation value is set.

### Steps to reproduce

add tilemap node,set tileset,click tile image set the animation properties.

**Comment by groud** [comment]
> Separation unit should not be px,It is clearly misleading.

That's true, it's a mistake.

> Some operations require order, otherwise nothing will happen. some hints should be added to this.

I don't know what to do about it to be honest. Maybe we can pop up an error message but that seems a little bit intrusive.
Values that would lead to tiles outside of the atlas are simply not applied for now, but maybe we could keep the values and ignore the invalid frames instead. That would require a warning somewhere in the tile inspector I suppose.

> Clear Array nothing happened. 

I guess this comes from the fact the array cannot have less than one element. I am not sure how to solve it though.

> Click Frames is a certain probability crash directly and exit to the desktop, after separation value is set.

It will need a stacktrace I guess.

**Comment by CodeLazier** [comment]
I don't know what to do about it to be honest. Maybe we can pop up an error message but that seems a little bit intrusive. Values that would lead to tiles outside of the atlas are simply not applied for now, but maybe we could keep the values and ignore the invalid frames instead. That would require a warning somewhere in the tile inspector I suppose.
> there may be modeless pop-up window? I remember godot4 had a warning pop-up message.

I guess this comes from the fact the array cannot have less than one element. I am not sure how to solve it though.
> Can this menu option be disabled?

Click Frames is a certain probability crash directly and exit to the desktop, after separation value is set.
> If it crash again, I'll keep an eye out.

**Comment by hsandt** [comment]
In my version v4.0.beta.custom_build [767f8fb48] I don't see px for Separation anymore. The workflow is still quite unclear though, but from what I understand, you must set the number of "Columns" to N and then define N durations.

Shouldn't we sync the number of Duration entries with Columns value then? (and also put both near each other in the interface) Is there an interest in keeping Durations independent, such as reusing the last duration for any extra tile? (but that's not too intuitive... I would understand if using a single, uniform duration for *all* tiles though)

And since separation can be on X or Y, we could rename Columns "Frames" or "Frames count", I suppose.

**Comment by vmedea** [comment]
Using 4.0beta5, I noticed that it doesn't seem to be possible to change a frame duration for a group of selected tiles at once. I select all the tiles, change duration from say, 1 to 0.5 second, then only the first selected tile changes. Am I missing something? I want all the animations to be in sync, but can't imagine having to change this for every tile individually!

FWIW: The number of frames is the same for every tile.

In my case I was able to work around by increasing the speed (which does update for all selected tiles). But I don't think it's intended behavior.
